include/dsn/c/api_layer1.h

```diff
@@ -558,7 +558,8 @@ typedef union dsn_msg_context_t
     struct {
         uint64_t is_request : 1;           ///< whether the RPC message is a request or response
         uint64_t is_forwarded : 1;         ///< whether the msg is forwarded or not
-        uint64_t unused : 4;               ///< not used yet
+        uint64_t is_reset_network : 1;     ///< set the flags means that the underlying network is reset
```

Comment:
This concept is not related to messages - you may use a special RPC task code instead for this purpose.